ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/outputconsole/OutputConsole.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.outputconsole;
+
+import org.eclipse.che.ide.api.mvp.Presenter;
+
+/**
+ * Describes requirements for the console for displaying output. 
+ *
+ * @author Roman Nikitenko
+ */
+public interface OutputConsole extends Presenter {
```

Comment:
It's a part of IDE API. So it has to be placed into che-core-ide-api module in order to avoid extensions depend on che-core-ide-app.
@vparfonov WDYT?